src/views/Biobanker.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <navigation-bar />
```

Comment:
There's no need for a navigation bar here, probably we need to change the App.vue